src/pages/ReimbursementAccount/ReimbursementAccountPage.tsx

```diff
@@ -257,7 +257,9 @@ function ReimbursementAccountPage({route, policy, isLoadingPolicy}: Reimbursemen
             return;
         }
 
-        setReimbursementAccountLoading(true);
+        if (policyIDParam) {
```

Comment:
Can we just return early here if there is no `policyIDParam`? 
We don't need to call clearReimbursementAccountDraft and fetchData won't be called if there is no policyID. Or do we still need `isStepToOpenEmpty` check in this case?

Reply:
I don't follow why we don't need to call `clearReimbursementAccountDraft`. `isStepToOpenEmpty` is not related to the policyID param, so I would leave it as it is. We can update the code to this if needed:

```
...
const isStepToOpenEmpty = getStepToOpenFromRouteParams(route) === '';
if (isStepToOpenEmpty) {
  setBankAccountSubStep(null);
  setPlaidEvent(null);
}

if (!policyIDParam) {
  return;
}
setReimbursementAccountLoading(true);
fetchData();
```